src/Events/Admin/Onboarding/Controller.php

```diff
@@ -88,6 +88,53 @@ public function handle_onboarding_page_dismiss(): void {
 		$this->container->make( Landing_Page::class )->handle_onboarding_page_dismiss();
 	}
 
+	/**
+	 * Redirects users to the Guided Setup page when accessing any TEC settings or management page for the first time.
+	 *
+	 * @since TBD
+	 *
+	 * @return void
+	 */
+	public function redirect_tec_pages_to_guided_setup(): void {
+		// Get some information about the page that we are trying to access.
+		$page      = tribe_get_request_var( 'page' );
+		$post_type = tribe_get_request_var( 'post_type' );
```

Comment:
I would use the `tec_` prefix where available, so we move away from `tribe_` where we can.